logger.go

```diff
@@ -321,3 +321,27 @@ func (logger *Logger) AddHook(hook Hook) {
 	defer logger.mu.Unlock()
 	logger.Hooks.Add(hook)
 }
+
+func (logger *Logger) IsDebugEnabled() bool {
```

Comment:
I'd rather have a single method `IsLogLevelEnabled` which takes a Level parameter, there would be less methods to implement, to document and no modification would be required if someday we add a new trace level.

Reply:
`IsXxxxEnabled` refactored to `IsLevelEnabled`